js/views/modals/purchase/Purchase.js

```diff
@@ -452,6 +452,7 @@ export default class extends BaseModal {
         singleSelect: true,
         selectFirst: true,
         radioStyle: true,
+        useCache: false,
```

Comment:
Sorry coming in late here. We are talking about waiting for mods to load in when purchasing? I really thinking waiting kind of sucks but picking bad moderators was a huge suck on 1.x. Getting latest info might actually be best here.